### ISSUE TITLE
[HOLD for payment 2023-03-09] Profile page - More horizontal padding then other pages

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#14745]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. Open app
2. Go to profile page

## Expected Result:
Horizontal padding should be consistent with other pages

## Actual Result:
Has more horizontal padding then other pages

## Workaround:
unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android / native
- [x] Android / Chrome
- [x] iOS / native
- [x] iOS / Safari
- [x] MacOS / Chrome / Safari
- [x] MacOS / Desktop

**Version Number:** 1.2.63-0
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 
<img width="960" alt="1" src="https://user-images.githubusercontent.com/43996225/216205563-cf29bab7-87f8-4457-9797-310da7033c7d.png">
<img width="372" alt="Screenshot 2023-02-01 at 10 01 17 AM" src="https://user-images.githubusercontent.com/43996225/216205592-55e93e80-17b9-4ad4-b023-73963713cc56.png">

**Expensify/Expensify Issue URL:**
**Issue reported by:** @shawnborton 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1675236178667749

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01f8c06b3565752fb8</li>
        <li>Upwork Job ID: 1621112944905035776</li>
        <li>Last Price Increase: 2023-02-02</li>
    </ul>
</details>

**Comment by cristipaval** [comment]
on it today

**Comment by MelvinBot** [comment]
@cristipaval, @kevinksullivan, @aimane-chnaif Whoops! This issue is 2 days overdue. Let's get this updated quick!

**Comment by MelvinBot** [comment]
@cristipaval, @kevinksullivan, @aimane-chnaif Whoops! This issue is 2 days overdue. Let's get this updated quick!

**Comment by cristipaval** [comment]
not overdue.

**Comment by MelvinBot** [comment]
@cristipaval, @kevinksullivan, @aimane-chnaif Uh oh! This issue is overdue by 2 days. Don't forget to update your issues!

**Comment by kevinksullivan** [comment]
Still in review

**Comment by MelvinBot** [comment]
@cristipaval, @kevinksullivan, @aimane-chnaif Uh oh! This issue is overdue by 2 days. Don't forget to update your issues!

**Comment by MelvinBot** [comment]
`Reviewing` label has been removed, please complete the "BugZero Checklist".

**Comment by MelvinBot** [comment]
The solution for this issue has been :rocket: deployed to production :rocket: in version 1.2.77-4 and is now subject to a 7-day regression period :calendar:. Here is the list of pull requests that resolve this issue:

- https://github.com/Expensify/App/pull/14855


If no regressions arise, payment will be issued on 2023-03-09. :confetti_ball:

After the hold period is over and BZ checklist items are completed, please complete any of the applicable payments for this issue, and check them off once done.
- [ ] External issue reporter
- [ ] Contributor that fixed the issue
- [ ] Contributor+ that helped on the issue and/or PR

As a reminder, here are the bonuses/penalties that should be applied for any External issue:
- Merged PR within 3 business days of assignment - 50% **bonus**
- Merged PR more than 9 business days after assignment - 50% **penalty**

**Comment by MelvinBot** [comment]
**BugZero Checklist:** The PR fixing this issue has been merged! The following checklist ([instructions](https://stackoverflow.com/c/expensify/questions/14574)) will need to be completed before the issue can be closed:

- [ ] [@aimane-chnaif / @cristipaval] The PR that introduced the bug has been identified. Link to the PR:
- [ ] [@aimane-chnaif / @cristipaval] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment:
- [ ] [@aimane-chnaif / @cristipaval] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion:
- [ ] [@aimane-chnaif] Propose [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) steps to ensure the same bug will not reach production again.
- [ ] [@kevinksullivan] Link the GH issue for creating/updating the regression test once above steps have been agreed upon:

**Comment by kevinksullivan** [comment]
@aimane-chnaif I think this regression test is pretty straightforward, right?

1. Open app
2. Go to profile page, observe padding between menu items
3. Go to preferences pages, observe padding between menu items
4. Confirm horizontal padding should be consistent with other pages

Going to go with this. Can you complete the remaining steps above so we can close this out @cristipaval @aimane-chnaif ?

**Comment by cristipaval** [comment]
It was a follow-up PR on [this one](https://github.com/Expensify/App/pull/13290)

**Comment by cristipaval** [comment]
Oh wait, it looks like it wasn't a follow-up, but a bug introduced by [that PR](https://github.com/Expensify/App/pull/13290).

**Comment by cristipaval** [comment]
@kevinksullivan I just checked the remeaining steps.

**Comment by kevinksullivan** [comment]
Thanks everyone. Just an update on payment @aimane-chnaif . For whatever reason the original post says it's closed, and upwork isn't letting me recreate the job at the moment... I will have to check back on this Monday as I'm signing off shortly. Thanks for your patience!

**Comment by cristipaval** [comment]
not overdue. @kevinksullivan is taking care of this today.

**Comment by cristipaval** [comment]
on @kevinksullivan

**Comment by cristipaval** [comment]
Melvin, you ok? Just remove it, no need to instantly add it again.

**Comment by cristipaval** [comment]
Hahah, yeah, I think @kevinksullivan was swamped at the ECX last week and that's why he missed this one.

**Comment by kevinksullivan** [comment]
Hey @aimane-chnaif @cristipaval ! So sorry for the delay. Cristi is right - ECX was a lot, and then I got sick on my return and have been OOO, which slowed me down more 😅 ... 

@aimane-chnaif I had to recreate the post and invited you to it. Can you let me know once you've accepted?

https://www.upwork.com/jobs/~019226bf24e884abee

**Comment by kevinksullivan** [comment]
Sorry @aimane-chnaif , withdrew one offer so there should be one now.

**Comment by kevinksullivan** [comment]
All set
